Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/GraphSerialization.cpp

```diff
@@ -145,7 +145,9 @@ namespace ScriptCanvas
                 , AZ::ObjectStream::FilterDescriptor(nullptr, AZ::ObjectStream::FILTERFLAG_IGNORE_UNKNOWN_CLASSES)))
             {
                 result.m_isSuccessful = false;
-                result.m_errors = "JSON (and failsafe XML) deserialize attempt failed.";
+                result.m_errors = "JSON (and failsafe XML) deserialize attempt failed.\n";
+                result.m_errors += "The error might be caused by deprecated Spawnable Nodes in your target scriptcanvas file.\n";
+                result.m_errors += "If so, please run UpdateSpawnableNodes python script for the scriptcanvas file to see if the error is resolved.";
```

Comment:
The location of where to find the script and instructions on how to run should also be provided. 

Also, it might be good to get Script canvas developers' eyes on this PR since we are adding explicit system specific(spawnables) error messages during loading any SC file.

@lsemp3d and @carlitosan , fyi.

Reply:
Messages updated.